mlflow/server/js/src/experiment-tracking/components/MetricsPlotControls.js

```diff
@@ -11,6 +11,13 @@ export const X_AXIS_STEP = 'step';
 export const X_AXIS_RELATIVE = 'relative';
 export const MAX_LINE_SMOOTHNESS = 100;
 
+const styles = {
```

Comment:
We usually define styles constant after the component class. It is safe because `render` executes after the initial file loading.